lib/utils/Kconfig

```
@@ -57,4 +57,10 @@ config UTF8
 	  Enable the utf8 API. The API implements functions to specifically
 	  handle UTF-8 encoded strings.
 
+config COBS
+	bool "Consistent overhead byte stuffing"
+	imply NET_BUF
```

Your implementation needs `NET_BUF` to work.

```suggestion
	select NET_BUF
```